src/test/java/teammates/test/cases/browsertests/StudentFeedbackSubmitPageUiTest.java

```diff
@@ -594,6 +594,21 @@ private void testInputValidation() {
         submitPage.fillResponseRichTextEditor(2, 2, "Response to no recipient");
         submitPage.submitWithoutConfirmationEmail();
         submitPage.verifyStatus("You did not specify a recipient for your response in question 2.");
+
+        ______TS("cannot choose self when generating choices from students (excluding self)");
+        submitPage = loginToStudentFeedbackSubmitPage("Alice", "Open Session");
+        assertFalse(submitPage.doesMcqOptionExist(24, 0, "Alice Betsy</option></td></div>'\" (Team >'\"< 1</td></div>'\")"));
+        assertTrue(submitPage.doesMcqOptionExist(24, 0, "Charlie Davis (Team 2)"));
+        submitPage.submitWithoutConfirmationEmail();
```

Comment:
Do you need to submit twice for this test case? i.e. can do the assertions for both question 24/25 (or 25/26 I'm not sure if it is 0- or 1-indexed) together before submitting?

Reply:
Yup can! Some part of the code on top submitted for every question and some submitted after a few questions so I wasn't sure which one to follow. Changed so that only one submission is required.